libsandbox/libsandbox.c

```diff
@@ -141,7 +141,10 @@ int resolve_dirfd_path(int dirfd, const char *path, char *resolved_path,
 		 */
 		sprintf(resolved_path, "%s/%i", sb_get_fd_dir(), dirfd);
 	}
-	ssize_t ret = readlink(resolved_path, resolved_path, at_len);
+
+
+	char buffer[at_len];
```

Comment:
VLA again...

Reply:
See my comment to the patch of erealpath(.) above.